Automattic/wp-calypso
# Enable RUM tracking in prod to 5% users

#### Changes proposed in this Pull Request

* Enable RUM tracking in production to 5% users

#### Testing instructions

* Load calypso and enable abtest debugging (run `localStorage.setItem('debug', 'calypso:abtests');`), more info in https://github.com/Automattic/wp-calypso/tree/master/client/lib/abtest
* Verify you get assigned the variation `noData` (or use the abtest helper button to assign it yourself). Verify there are no requests to `/logstash` when you navigate between 'My Home', 'Stats' or 'Plan'.
* Assign yourself the variation `collectData`, navigate between those pages and verity there are calls to `/logstash`
